### PR TITLE
Fix issue #614

### PR DESCRIPTION
**Pull request by poormusic2001**
as title

Fix issue #614

Catalyst: custom background not shown after launch of monal #614

### CODE REVIEW

File: Monal/Classes/MLPlaceholderViewController.h [hunk 1]
```diff
@@ -7,10 +7,13 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "HelperTools.h"
+#import "MLImageManager.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface MLPlaceholderViewController : UIViewController
+@property (weak, nonatomic) IBOutlet UIImageView* backgroundImageView;
```
Comment (FriedrichAltheide) [inline Monal/Classes/MLPlaceholderViewController.h]: Does it need to be public?

File: Monal/Classes/MLPlaceholderViewController.m [hunk 2]
```diff
@@ -17,14 +17,38 @@ @implementation MLPlaceholderViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
+    
+    NSNotificationCenter* nc = [NSNotificationCenter defaultCenter];
+    [nc addObserver:self selector:@selector(updateBackground) name:kMonalBackgroundChanged object:nil];
```
Comment (FriedrichAltheide) [inline Monal/Classes/MLPlaceholderViewController.m:24]: corresponding removeObserver is missing

File: Monal/Classes/MLPlaceholderViewController.m [hunk 3]
```diff
@@ -17,14 +17,38 @@ @implementation MLPlaceholderViewController
...
 
 -(void) viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
     self.splitViewController.preferredDisplayMode=UISplitViewControllerDisplayModeAllVisible;
+    
+    [self updateBackground];
 }
 
-
+-(void) updateBackground {
```
Comment (FriedrichAltheide) [inline Monal/Classes/MLPlaceholderViewController.m]: code style

File: Monal/Classes/MLPlaceholderViewController.m [hunk 4]
```diff
@@ -17,14 +17,38 @@ @implementation MLPlaceholderViewController
...
+    BOOL backgrounds = [[HelperTools defaultsDB] boolForKey:@"ChatBackgrounds"];
+    
+    if(backgrounds){
```
Comment (FriedrichAltheide) [inline Monal/Classes/MLPlaceholderViewController.m]: code style (all if)

File: Monal/Classes/MLPlaceholderViewController.m [hunk 5]
```diff
@@ -17,14 +17,38 @@ @implementation MLPlaceholderViewController
...
+        self.backgroundImageView.hidden = NO;
+        NSString* imageName = [[HelperTools defaultsDB] objectForKey:@"BackgroundImage"];
+        if(imageName)
```
Comment (FriedrichAltheide) [inline Monal/Classes/MLPlaceholderViewController.m:50]: I would prefer a debug assert if imageName is nil
@tmolitor-stud-tu "BackgroundImage" should always have a entry correct?

File: Monal/localization/Base.lproj/Main.storyboard [hunk 6]
```diff
@@ -328,6 +328,9 @@
                             <constraint firstAttribute="bottom" secondItem="mSj-cI-xo2" secondAttribute="bottom" constant="29" id="tMp-Od-ics"/>
                         </constraints>
                     </view>
+                    <connections>
```
Comment (FriedrichAltheide) [inline Monal/localization/Base.lproj/Main.storyboard:331]: Please remove the current default image from the storyboard

**Review by FriedrichAltheide — CHANGES_REQUESTED** [review]

File: Monal/Classes/MLPlaceholderViewController.m (hunk 5 above)
Comment (poormusic2001) [reply Monal/Classes/MLPlaceholderViewController.m:50]: If imageName is nil, I let backgroundImageView to hide itself. 

So I add a else to hide the backgroundImageView.

**Pull request merged**